src/ClusterManager/job_launcher.py

```diff
@@ -786,6 +806,9 @@ def kill_job(self, job_id, desired_state="killed"):
             logger.error("Kill job failed with errors: {}".format(errors))
             return False
 
+    def scale_job(self, job):
+        pass
```

Comment:
Maybe add a logger.warning to show controller doesn't support it now.

Reply:
This will dump too many logs, just skip it now.